po/es.po

```diff
@@ -0,0 +1,606 @@
+#. Name of the perspective
+#: assets/ui/media-toc.ui:1598 assets/ui/media-toc.ui:1979
+msgid "Split"
+msgstr "Dividir"
+
+#. Button tooltip
+#: assets/ui/media-toc.ui:1602
+msgid "Split audio stream and encode to selected format"
+msgstr "Dividir la secuencia de audio y codificarla al formato seleccionado"
+
+#: assets/ui/media-toc.ui:1620
+msgid "Split Formats"
+msgstr "Formatos de corte"
+
+#. Name of the perspective
+#: assets/ui/media-toc.ui:1794
+msgid "Display"
+msgstr "Visualización"
+
+#. Name of the perspective
+#: assets/ui/media-toc.ui:1856
+msgid "Streams"
+msgstr "Flujos"
+
+#. Name of the perspective
+#: assets/ui/media-toc.ui:1918
+msgid "Export\t"
+msgstr "Exportar"
+
+#: src/application/command_line.rs:13 src/ui/main_controller.rs:266
+msgid ""
+"Build a table of contents from a media file\n"
+"or split a media file into chapters"
+msgstr ""
+"Construye una tabla de contenido a partir de un archivo multimedia\n"
+"o divide un archivo multimedia en capítulos"
+
+#: src/application/command_line.rs:14
+msgid "Display this message"
+msgstr "Muestra este mensaje"
+
+#: src/application/command_line.rs:15
+msgid "Print version information"
+msgstr "Muestra la versión"
+
+#: src/application/command_line.rs:18
+msgid "MEDIA"
+msgstr "ARCHIVO MULTIMEDIA"
+
+#: src/application/command_line.rs:30
+msgid "Disable video rendering hardware acceleration"
+msgstr ""
+
+#: src/application/command_line.rs:34
+msgid "Path to the input media file"
+msgstr "Ruta al archivo multimedia"
+
+#: src/application/configuration.rs:63
+msgid "couldn't load configuration: {}"
+msgstr "No se pudo cargar la configuración: {}"
+
+#: src/application/configuration.rs:100
+msgid "couldn't write configuration: {}"
+msgstr "No se pudo escribir la configuración: {}"
+
+#: src/application/configuration.rs:111
+msgid "couldn't serialize configuration: {}"
+msgstr "No se pudo serializar la configuración: {}"
+
+#: src/application/configuration.rs:123
+msgid "couldn't create configuration file: {}"
+msgstr "No se pudo crear el archivo de configuración: {}"
+
+#: src/main.rs:52
+msgid "Failed to initialize GTK"
+msgstr "No se pudo inicializar GTK"
+
+#: src/media/playback_context.rs:65
+msgid "Opening {}..."
+msgstr "Abriendo {}..."
+
+#: src/media/playback_context.rs:87
+msgid ""
+"Missing `decodebin3`\n"
+"check your gst-plugins-base install"
+msgstr ""
+"Falta `decodebin3`\n"
+"Verifique la instalación de gst-plugins-base"
+
+#: src/media/playback_context.rs:104
+msgid "Couldn't find GStreamer GTK video sink."
+msgstr "No se pudo encontrar el elemento \"GTK video sink\""
+
+#: src/media/playback_context.rs:105
+msgid "Video playback will be disabled."
+msgstr "La reproducción de vídeo estará deshabilitada."
+
+#: src/media/playback_context.rs:106
+msgid "Please install {} or {}, depending on your distribution."
+msgstr "Por favor installe {} o {} en función de su distribución."
+
+#: src/media/playback_context.rs:132 src/media/toc_setter_context.rs:74
+#, fuzzy
```

Comment:
This one is still marked as `fuzzy`. If you're happy with current translation, just add a space (or anything else) and then remove it so that the editor knows it's OK. I think you used a different wording for "Playing" elsewhere though (like in the tooltip for the "Play / Pause" button).

Reply:
ok, i'm happy with the translation so I add/remove a space. Thanks!